config/foreman.migrations/20250930000666_remove_grub1.rb

```diff
@@ -0,0 +1,7 @@
+if (proxy = answers['foreman_proxy'])
```

Comment:
You need to check if it's a hash because it can be a boolean. It's very unlikely to happen in our scenario, but I strongly believe in building up good patterns.
```suggestion
if answers['foreman_proxy'].is_a?(Hash)
  proxy = answers['foreman_proxy']
```